src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,71 @@
+.calendar {
+  padding: 10px;
+  width: 706px;
+  display: flex;
+  flex-flow: row wrap;
+  gap: 1px;
+
+  @for $i from 1 through 31 {
+    .calendar__day:nth-child(#{$i}) {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      box-sizing: border-box;
+      background-color: $cell-color;
+      width: $cell-size;
+      height: $cell-size;
+      border: 1px solid black;
```

Comment:
```suggestion

```
Consider moving these styles to a basic class instead of generating them for each element in the cycle